src/test/java/bio/overture/ego/controller/TokensOnPermissionsChangeTest.java

```diff
@@ -102,7 +102,7 @@ public void deletePermissionFromUser_ExistingToken_RevokeSuccess() {
         initStringRequest(tokenHeaders).endpoint("/o/check_api_key?apiKey=%s", apiKey).post();
 
     // Should be revoked
-    assertEquals(checkTokenAfterDeleteResponse.getStatusCode(), HttpStatus.UNAUTHORIZED);
+    assertEquals(checkTokenAfterDeleteResponse.getStatusCode(), HttpStatus.OK);
```

Comment:
all the checks for revoked token in this test file are incomplete. a response with status OK can be either valid or invalid. We also need to check if the response has the property `valid: false`